datastream-utils/src/test/java/com/linkedin/datastream/common/TestLogUtils.java

```diff
@@ -31,4 +31,23 @@ public void testLogNumberArrayInRange() throws Exception {
 
     Assert.assertEquals(LogUtils.logNumberArrayInRange(null), "[]");
   }
+
+  @Test
+  public void testLogSummarizedTopicPartitionsMapping() throws Exception {
+    List<String> input = Arrays.asList("t-1", "t-3", "t-6", "t-4", "t-7", "t-12", "t-9", "t-10", "t-8");
```

Comment:
could there be an actual case like that? 
> Eg, "t-0, t-1, t--2" --> "t:[0-2]"
Since reusing this [code](https://github.com/linkedin/brooklin/blob/master/datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/TopicPartitionUtil.java) would give output like,
> "t:[0-1], t-:[2]"

Reply:
the topic name can be topic-abc-def-ghi-xyz